src/attr/_make.py

```diff
@@ -669,24 +669,29 @@ def _create_slots_class(self):
             if k not in tuple(self._attr_names) + ("__dict__", "__weakref__")
         }
 
-        # Traverse the MRO to check for an existing __weakref__ and
-        # __setattr__.
-        custom_setattr_inherited = False
+        # Check the bases if one of them has an attrs-made __setattr__ that
```

Comment:
I've went for:

```
        # If our class doesn't have an own __setattr__ (either from the user or
        # by us), check the bases, if one of them has an attrs-made
        # __setattr__, that needs to be reset.
```

Is that clear & OK?

Reply:
I am slightly confused by "have an own `__setattr__`", because "an own X" is not a common idiom in my dialect of English (can't speak to BrE or European flavors), though I recognize that it's something of a term of art in this library. That said, the "If one of them has an attrs-made `__setattr__` that needs to be reset" clause removes all ambiguity, so it's not a big deal.

I would probably replace "have an own `__setattr__`" with "have its own `__setattr__`" or even "have its own implementation of `__setattr__`", but I don't think leaving it as-is would meaningfully impact my ability to understand the comment.

Reply:
Perfect!